pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -157,6 +158,19 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		r.restMapper,
 	)
 	if err != nil {
+		if err == errServiceNotFound {
+			conditionsv1.SetStatusCondition(&sbr.Status.Conditions, conditionsv1.Condition{
+				Type:    BindingReady,
+				Status:  corev1.ConditionFalse,
+				Reason:  ServiceNotFoundReason,
```

Comment:
We can have a slice of services for multiple backing services, wherein we know which service is found and which service is not found, and then flag the readiness as False

Reply:
Kind of like we have for applications?

```
status:
  applications:
  - group: apps
    kind: Deployment
    name: nodejs-rest-http-crud
    version: v1
```